Source/WebGPU/WebGPU/WebGPUExt.h

```diff
@@ -177,12 +177,15 @@ WGPU_EXPORT void wgpuDeviceClearUncapturedErrorCallback(WGPUDevice device) WGPU_
 #define HAS_SWIFTCXX_THUNK  NS_REFINED_FOR_SWIFT
 
 #if defined(ENABLE_WEBGPU_SWIFT) && ENABLE_WEBGPU_SWIFT && defined(__WEBGPU__)
+#include <Metal/Metal.h>
```

Comment:
why do we need to include these headers into the bottom of a header file?

Reply:
IIRC the headers need some stuff declared in WebGPUExt.h.

Reply:
Shouldn't WebGPUExt.h be included in the headers then and not the other way around?

Reply:
Yea this header list is not in the right place and I am planning to clean this up in:
https://github.com/WebKit/WebKit/pull/36998
(Note that above PR is not complete yet so I will have another look once I am done with this one)